### ISSUE TITLE
status

### ISSUE THREAD

**Issue by chemuto** [linhua55/twitter_smali#1]
Hi, I found your repository here: [https://github.com/Fuzion24/JustTrustMe/issues/17](url) 
I tried to compile this code and run it on a genymotion emulator with api 24 but the twitter app crashes.

Is it working for you?

Thanks

**Comment by linhua55** [comment]
Yes, it works well on my android phone.  The genymotion does not work well with app which have arm architecture binary code.

I use Android-killer to decompile the twitter apk file.    You  can use [Apktool](https://ibotpeaches.github.io/Apktool/) to recompile this to apk file.
The Android-killer is a tool which wraps/automates some other command line tools like  [ShakaApktool ](https://github.com/rover12421/ShakaApktool) which is a fork of Apktool to support Chinese character(method name etc.) .

http://www.jianshu.com/p/19b62aace407
http://down.52pojie.cn/Tools/Android_Tools/
